src/utils.lisp

```diff
@@ -44,9 +44,10 @@
           (car vals-tail) next-val       ; Must... resist... the urge...
           (cdr vals-tail) (car alist)))) ; Gaaaah!
 
-(defmacro fun (&body body)
-  "This macro puts the FUN back in FUNCTION."
-  `(lambda (&optional _) (declare (ignorable _)) ,@body))
+(eval-when (:compile-toplevel :load-toplevel :execute)
```

Comment:
eval-always is never a good idea, so I prefer PRs with as few of these as possible.